lib/Doctrine/ruleset.xml

```diff
@@ -410,6 +410,10 @@
     </rule>
     <!-- Forbid global functions -->
     <rule ref="Squiz.Functions.GlobalFunction"/>
+    <!-- Force camelCase variables -->
+    <rule ref="Squiz.NamingConventions.ValidVariableName">
+        <exclude name="Squiz.NamingConventions.ValidVariableName.PrivateNoUnderscore" />
```

Comment:
Why excluding this one?

Reply:
It forces private properties to start with `_` 🙄